test/message/test_runner_output.js

```diff
@@ -61,6 +61,11 @@ test('async assertion fail', async () => {
   assert.strictEqual(true, false);
 });
 
+test('async assertion object fail', async () => {
```

Comment:
Would the diff of this PR be smaller if you added this test at the end of the file?

Reply:
sure